examples/mnist/mnist.go

```diff
@@ -1,21 +1,23 @@
 package main
 
 import (
-	"fmt"
-
 	torch "github.com/wangkuiyi/gotorch"
 )
 
-func main() {
+func trainLoop() {
 	dataset := torch.NewMNIST("./data")
 	dataset.AddTransforms([]torch.Transform{
 		torch.NewNormalize(0.1307, 0.3081),
 		torch.NewStack(),
 	})
 	trainLoader := torch.NewDataLoader(dataset, 8)
 	for trainLoader.Scan() {
-		data := trainLoader.Data()
-		fmt.Println(data.Data)
-		fmt.Println(data.Target)
+		torch.GC()
```

Comment:
Maybe we can consider putting the call to `torch.GC()` in `DataLoader.Scan()` later.

Reply:
It's a good idea,  but we also need to call `dataset.Close()` and `dataloader.Close()`, maybe we can move these to `torch.FinishGC()`, that the train loop can be more clear:

``` golang
trainLoader := torch.NewDataLoader(dataset, 8)
for trainLoader.Scan() {
	trainLoader.Data()
}
torch.FinishGC()
```